deploy/kube-config/kubedash/heapster-controller.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
is it possible to post incremental changes as separate commits and then squash them at the end? Github makes it hard to track comments if commits are rebased :(